Moonscraper Chart Editor/Assets/Scripts/Game/UI/LyricEditor2/LyricEditor2PhraseController.cs

```diff
@@ -125,12 +125,12 @@ public void SetPhraseEnd(uint tick)
 
     public void PickupPhraseStart() 
     {
-        phraseStartEvent.Pickup().Invoke();
+        phraseStartEvent.Pickup()?.Invoke();
```

Comment:
This needs further investigation. I'd recommend pulling the lyric editor fix into a separate PR so that this isn't blocking the other two fixes while I look into what's causing this. Need to figure out if null is intentionally meant to be valid or not.

Reply:
Will do!